### DISCUSSION TITLE
Error in h(simpleError(msg, call))

### DISCUSSION THREAD

**Discussion by nmiddegh** [metrumresearchgroup/mrgsolve#836] [Q&A] [answered]
Script:
source("src/global.R")
library(dplyr)
library(mrgsolve)
rm(list = ls())
mod <- mread("azithro", "model") %>% zero_re

Error:
Error in h(simpleError(msg, call)) : 
  error in evaluating the argument '.x' in selecting a method for function 'zero_re': project directory 'model' must exist and be readable.

I am getting this error when I am trying compile the model. Can someone help me to fix this issue. Thanks a lot. -NM

**Comment by kylebaron** [comment, marked as answer]
Hi @nmiddegh - 

It looks like the error message is a little complicated by the piping that  is going on. It looks like `mread()` can't find the `azithro.cpp` file.  It is looking for a directory called `model` but can't find it. Can you check to make sure the `model` directory exists (from your R working directory) and that the `azithro.cpp` file is there? Otherwise, you should point `mread()` to the location where the file is. 

Let me know your progress on this. 

Thanks, 
Kyle

BTW: I moved your issue to our discussions area; you will see it here
https://github.com/metrumresearchgroup/mrgsolve/discussions
rather than the issue tracker where you originally posted.

**Reply by nmiddegh** [reply]
Thank you Kyle. You are right. It is working after I changed the working directory. 

Best,
Narasimha
